src/signup-exports/lambda/lambda.ts

```diff
@@ -3,17 +3,31 @@ import * as SSM from 'aws-sdk/clients/ssm';
 import { createDatabaseConnectionPool } from '../../lib/db';
 import { getDatabaseParamsFromSSM } from '../../lib/ssm';
 import { uploadAsCsvToS3 } from '../../lib/upload';
-import { getCreatedOrCancelledSignupsFromYesterday } from '../lib/db';
+import {
+	getCreatedOrCancelledOneOffSignupsFromYesterday,
+	getCreatedOrCancelledRecurringSignupsFromYesterday,
+} from '../lib/db';
 import { getYesterday } from '../lib/utils';
 
 const ssm: SSM = new AWS.SSM({ region: 'eu-west-1' });
 
 const yesterday = getYesterday();
-const S3_KEY = `one-off-signups/date=${yesterday}/one-off-signups.csv`;
+const ONE_OFF_S3_KEY = `one-off-signups/date=${yesterday}/one-off-signups.csv`;
+const RECURRING_S3_KEY = `recurring-signups/date=${yesterday}/recurring-signups.csv`;
 const S3_BUCKET = process.env.Bucket ?? '';
 
-export const handler = async (): Promise<number> =>
-	getDatabaseParamsFromSSM(ssm)
-		.then(createDatabaseConnectionPool)
-		.then(getCreatedOrCancelledSignupsFromYesterday)
-		.then((result) => uploadAsCsvToS3(result, S3_BUCKET, S3_KEY));
+export const handler = async (): Promise<number> => {
+	const pool = await getDatabaseParamsFromSSM(ssm).then(
+		createDatabaseConnectionPool,
+	);
+
+	const numOneOffs = await getCreatedOrCancelledOneOffSignupsFromYesterday(
+		pool,
+	).then((result) => uploadAsCsvToS3(result, S3_BUCKET, ONE_OFF_S3_KEY));
+
+	const numRecurrings = await getCreatedOrCancelledRecurringSignupsFromYesterday(
+		pool,
+	).then((result) => uploadAsCsvToS3(result, S3_BUCKET, RECURRING_S3_KEY));
+
```

Comment:
it would be helpful to log both counts here, so we can easily find them in the cloudwatch logs

Reply:
we do log the requests made to the db, but it might be worth adding some extra `console.logs` in there to make it easier to find?
<img width="954" alt="Screenshot 2021-02-04 at 15 02 45" src="https://user-images.githubusercontent.com/17720442/106911556-30b5a580-66fa-11eb-914a-bb7914955b41.png">

Reply:
ah of course, never mind, that's plenty of logging!